### PR TITLE
Added 3 new Louisville micro-mobility feeds

### PR DESCRIPTION
**Pull request by schnuerle**

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -236,6 +238,7 @@ US,Sobi Long Beach,"Long Beach, NY",sobi_long_beach,http://sobilongbeach.com/,ht
 US,Spartanburg BCycle,"Spartanburg, SC",bcycle_spartanburg,https://spartanburg.bcycle.com,https://gbfs.bcycle.com/bcycle_spartanburg/gbfs.json
 US,Spin Chicago,"Chicago, IL",spin chicago,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/chicago/gbfs.json
 US,Spin Detroit,"Detroit, MI",spin detroit,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/detroit/gbfs
+US,Spin Louisville,"Louisville, KY",spin louisville,https://web.spin.pm/api/gbfs/v1/louisville/free_bike_status
```
Comment (jcn) [inline systems.csv]: This should point to the auto-discovery endpoint https://web.spin.pm/api/gbfs/v1/louisville/gbfs

File: systems.csv [hunk 2]
```diff
@@ -154,8 +154,10 @@ US,Bikeshare Kona,"Kona, HI",bikeshare_kona,https://hawaiiislandbikeshare.org/,h
 US,BIKETOWN,"Portland, OR",biketown_pdx,https://www.biketownpdx.com/,http://biketownpdx.socialbicycles.com/opendata/gbfs.json
 US,Biki,"Honolulu, HI",go_biki,https://gobiki.org/,https://hon.publicbikesystem.net/ube/gbfs/v1/
 US,Bird Chicago,"Chicago, IL",bird-chicago,https://www.bird.co,https://mds.bird.co/gbfs/chicago/gbfs.json
+US,Bird Louisville,"Louisville, KY",bird-louisville,https://www.bird.co,https://mds.bird.co/gbfs/louisville/free_bikes
 US,Bishop Ranch BRiteBikes,"San Ramon, CA",bishop_ranch,http://britebikes.socialbicycles.com/,http://britebikes.socialbicycles.com/opendata/gbfs.json
 US,Bolt Chicago,"Chicago, IL",bolt_chi,https://www.micromobility.com,https://www.bolt.miami/bolt2/chi/gbfs/gbfs.json
+US,Bolt Louisville,"Louisville, KY",bolt_lou,https://www.micromobility.com,https://www.bolt.miami/bolt2/lou/gbfs/gbfs.json
```
Comment (jcn) [inline systems.csv]: Let's have the system short name match what's in the feed: BOLT_LOU

File: systems.csv [hunk 3]
```diff
@@ -154,8 +154,10 @@ US,Bikeshare Kona,"Kona, HI",bikeshare_kona,https://hawaiiislandbikeshare.org/,h
 US,BIKETOWN,"Portland, OR",biketown_pdx,https://www.biketownpdx.com/,http://biketownpdx.socialbicycles.com/opendata/gbfs.json
 US,Biki,"Honolulu, HI",go_biki,https://gobiki.org/,https://hon.publicbikesystem.net/ube/gbfs/v1/
 US,Bird Chicago,"Chicago, IL",bird-chicago,https://www.bird.co,https://mds.bird.co/gbfs/chicago/gbfs.json
+US,Bird Louisville,"Louisville, KY",bird-louisville,https://www.bird.co,https://mds.bird.co/gbfs/louisville/free_bikes
```
Comment (jcn) [inline systems.csv]: This should point to the auto-discovery endpoint https://mds.bird.co/gbfs/louisville/gbfs.json

**Review by jcn — CHANGES_REQUESTED** [review]

File: systems.csv (hunk 1 above)
Comment (schnuerle) [reply systems.csv]: Agreed and fixed, sorry I missed it.

File: systems.csv (hunk 2 above)
Comment (schnuerle) [reply systems.csv]: Oh changed to that.

File: systems.csv (hunk 3 above)
Comment (schnuerle) [reply systems.csv]: Fixed and updated, thanks.

**Comment by schnuerle** [comment]
Made the recommended updates.

**Review by jcn — APPROVED** [review]

**Pull request merged**